schema/draft/schema.ts

```diff
@@ -1456,30 +1482,110 @@ export interface RootsListChangedNotification extends JSONRPCNotification {
   method: "notifications/roots/list_changed";
 }
 
+/**
+ * @category elicitation/create
+ */
+export interface ElicitRequestFormParams extends ElicitRequestParams {
+  /**
+   * The elicitation mode.
+   */
+  mode: "form";
+
+  /**
+   * A restricted subset of JSON Schema.
+   * Only top-level properties are allowed, without nesting.
+   */
+  requestedSchema: {
+    type: "object";
+    properties: {
+      [key: string]: PrimitiveSchemaDefinition;
+    };
+    required?: string[];
+  };
+}
+
+/**
+ * @category elicitation/create
+ */
+export interface ElicitRequestURLParams extends ElicitRequestParams {
+  /**
+   * The elicitation mode.
+   */
+  mode: "url";
+
+  /**
+   * The ID of the elicitation, which must be unique within the context of the server.
+   * The client MUST treat this ID as an opaque value.
+   */
+  elicitationId: string;
+
+  /**
+   * The URL that the user should navigate to.
+   *
+   * @format uri
+   */
+  url: string;
+}
+
+/**
+ * The parameters for a request to elicit additional information from the user via the client.
+ *
+ * @internal
+ */
+export interface ElicitRequestParams extends RequestParams {
+  /**
+   * The mode of elicitation.
+   * - "form": In-band structured data collection with optional schema validation
+   * - "url": Out-of-band interaction via URL navigation
+   */
+  mode: "form" | "url";
+
+  /**
+   * The message to present to the user.
+   * For form mode: Describes what information is being requested.
+   * For url mode: Explains why the interaction is needed.
+   */
+  message: string;
+}
+
 /**
  * A request from the server to elicit additional information from the user via the client.
  *
  * @category elicitation/create
  */
 export interface ElicitRequest extends JSONRPCRequest {
   method: "elicitation/create";
-  params: {
-    /**
-     * The message to present to the user.
-     */
-    message: string;
+  params: ElicitRequestFormParams | ElicitRequestURLParams;
+}
+
+/**
+ * The parameters for a request to track the progress of an elicitation.
+ *
+ * @category elicitation/track
+ */
+export interface ElicitTrackParams extends RequestParams {
+  _meta: {
     /**
-     * A restricted subset of JSON Schema.
-     * Only top-level properties are allowed, without nesting.
+     * The progress token as specified in the Progress capability, required in this request.
      */
-    requestedSchema: {
-      type: "object";
-      properties: {
-        [key: string]: PrimitiveSchemaDefinition;
-      };
-      required?: string[];
-    };
+    progressToken: ProgressToken;
+    [key: string]: unknown;
   };
+
+  /**
+   * The ID of the elicitation to track.
+   */
+  elicitationId: string;
+};
+
+/**
+ * A request from the client to track the progress of an elicitation.
+ *
+ * @category elicitation/track
+ */
+export interface ElicitTrackRequest extends JSONRPCRequest {
+  method: "elicitation/track";
+  params: ElicitTrackParams;
```

Comment:
Is this the only `JSONRPCRequest` in the schema that does not have a corresponding `Result` type? What result are you expected to produce a `JSONRPCResponse`? `{}`? We should probably specify and name the `Result` interface, but I think we should just remove tracking altogether. It's a bit underdefined, and the example branch in the typescript-sdk doesn't even implement it.